packages/miniflare/test/fixtures/assets/modules.ts

```diff
@@ -0,0 +1,35 @@
+import manifestJSON from "__STATIC_CONTENT_MANIFEST";
+import { getAssetFromKV } from "@cloudflare/kv-asset-handler";
```

Comment:
We should not be using `kv-asset-handler` in this fixture.

Reply:
indeed. This was a misunderstanding on my part wrt to how Workers Sites are set up. `kv-asset-handler` is not relevant to our assets implementation. Removed